src/loginPage/loginPage.component.tsx

```diff
@@ -226,13 +240,12 @@ export const AnonLoginScreen = (
 );
 
 export const LoginSelector = (
-  props: CombinedLoginProps & { mnemonics: ICATAuthenticator[] }
+  props: CombinedLoginProps & {
+    mnemonics: ICATAuthenticator[];
+    mnemonic: string | undefined;
+    setMnemonic: React.Dispatch<React.SetStateAction<string | undefined>>;
```

Comment:
Another small style comment. This feels overly complex - I'm sure the type `(mnemonic: string | undefined) => void` is probably fine